src/tools/bundle.ts

```diff
@@ -42,13 +42,19 @@ export async function bundle(entryPoints: string[], bundleName: string, verbose:
     }
     const archive = new JSZip();
     for (const file of listing) {
-      archive.file(file.bundlePath, fs.readFileSync(file.filePath));
+      archive.file(
+        file.bundlePath,
+        fs.readFileSync(file.filePath),
+        // Some unpackers have issues with 0-byte files.
+        {createFolders: false});
     }
     archive.file('bundle-manifest.mf', listing
         .filter(f => f.entryPoint)
         .map(f => `entry-point: ${f.bundlePath}\n`)
         .join(''));
-    archive.generateNodeStream({streamFiles: true})
+    // Don't use {streamFiles: true}.
+    // Java ZipInputStream does not acceptdata descriptors it generates.
```

Comment:
accept data

Reply:
but probably this is better still:

... does not accept the data descriptors that it generates.

Reply:
Done, Thank you!